assets/styles/_animation.scss

```diff
@@ -83,17 +149,3 @@ $length: 50%;
     opacity: 0.2;
   }
 }
-
-@keyframes arrowMove {
-  0% {transform: translateY(0)}
-  100% {transform: translateY(-4vh)}
-}
-
-@keyframes spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
```

Comment:
ser ut som det kan være noe her ?

Reply:
What do you mean? Is there something wrong? This is part of the code that I moved to the top of the same file.

Reply:
når jeg reviewet var den en rød sirkel der, som ikke synes her..som var på slutten av linja. men så ingen feil ;)

Reply:
Yeah, Github complains about a lack of a new line at the end of the file. You can see this if you hover over it. You can ignore this. 
![image](https://user-images.githubusercontent.com/54579944/164985200-784c96e8-12f4-419b-97df-bf024e4858e2.png)